x/smart-account/testutils/max_value_authenticator.go

```diff
@@ -26,6 +26,7 @@ type MaxAmountAuthenticator struct {
 }
 
 func (m MaxAmountAuthenticator) StaticGas() uint64 {
+	// Q: I see it's only used in tests but is it safe to leave as 0?
 	return 0
```

Comment:
Please commit if no concerns
```suggestion
	return 0
```